internal/common/mqtt/mqttconfig.go

```diff
@@ -48,6 +49,8 @@ type Message struct {
 
 var (
 	mqttClient *Client
+	clientID   string
```

Comment:
I think the `clientID` as a global variable is not a good approach. 
The `clientID` seems to be changed frequently by appID. I'm afraid a race condition problem when there are two requests from appID A and appID B at the same time.
I think you should use the `ID` of Client struct. What do you think?

Reply:
Done!!